inmoose/limma/toptable.py

```diff
@@ -156,23 +157,22 @@ def topTable(
 
     Returns
     -------
-    pd.DataFrame
+    DEResults or pd.DataFrame
         a dataframe with a row for the :code:`number` top genes and the
         following columns:
 
         - genelist: one or more columns of probe annotation, if
           :code:`genelist` was included as input
-        - logFC: estimate of the log2-fold-change corresponding to the effect
-          or contrast
+        - log2FoldChange: estimate of the log2-fold-change corresponding to the
+          effect or contrast
         - CI_L: left limit of confidence interval for :code:`logFC`, if
           :code:`confint=True` or :code:`confint` is a numeric value
         - CI_R: right limit of confidence interval for :code:`logFC`, if
           :code:`confint=True` or :code:`confint` is a numeric value
         - AveExpr: average log2-expression for the probe over all arrays and
           channels, same as :code:`Amean` in the :class:`MArrayLM` object
-        - t: moderated *t*-statistic
-        - F: moderated *F*-statistic
```

Comment:
If I remember correctly, we compute either F or t test depending on the results? Is there a way to know which stat have been computed?

Reply:
Exactly, *t*-test for a single contrast / coefficient, *F*-test otherwise.
When a *t*-test is performed, then a `DEResults` instance is returned, otherwise it's a `DataFrame`.
I should rephrase the docstring to explain this.